packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -567,13 +588,16 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
                 model,
                 treeNode
             }}>
-                <div title={tooltip} className='status' style={{ color }}>
+                <div title={tooltip} className={cName + ctype} style={{ color, backgroundImage }} >
                     {letter}
                 </div>
             </ScmInlineActions>
         </div >;
     }
 
+    protected isIcon(icon: URI | string | undefined): boolean {
+        return icon ? true : false;
+    }
```

Comment:
Not sure the method is necessary, we can directly do:

```ts
!!icon
```

But if you do want to keep the method we can simplify to:

```ts
protected isIcon(icon: URI | string | undefined): boolean {
    return !!icon;
}
```

Reply:
Please also add a newline after the method if we keep it.

Reply:
I was trying to achieve that, but did not consider the double negative. Thanks!
I don't like the method when you can easily do it with one variable.